packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -326,5 +330,7 @@ const Alert = ({
   );
 };
 
+const Alert = forwardRef(_Alert);
```

Comment:
Should we use that `assignWithoutSideEffects` and add displayNames to these components?

Reply:
I checked on storybook and they do seem to have correct display names there so don't think so its needed for other things